### PR TITLE
Release 97.0

### PR DESCRIPTION
**Pull request by matthiasblum**
Includes the following changes:

- Show basic information for deleted member database signatures #519
- Show AI-generated descriptions, when available #521
- Send annotation suggestions posted on Pfam pages to the Pfam helpdesk #522 
- Use Pfam's terminology (_clan_ instead of _set)_ on Pfam pages #523 
- Do not select representative domains in the client: use the new `representative` boolean property returned by the API #525 
- Fix loading state #527

### CODE REVIEW

File: src/components/SimpleCommonComponents/Tooltip/style.css [hunk 1]
```diff
@@ -10,6 +10,14 @@
   margin: 4px;
   z-index: var(--z-index-tooltips);
 
+  &.fixed {
```
Comment (gustavo-salazar) [inline src/components/SimpleCommonComponents/Tooltip/style.css:13]: was this trying to fix the overgrown tooltip? is it still necessary after my fix?

**Review by gustavo-salazar — APPROVED** [review]
I left some minor comments/questions, but nothing blocking this to get merge!

File: src/components/SimpleCommonComponents/Tooltip/index.tsx [hunk 2]
```diff
@@ -45,6 +48,14 @@ const Tooltip = ({
       arrow({
         element: arrowRef,
       }),
+      size({
```
Comment (gustavo-salazar) [inline src/components/SimpleCommonComponents/Tooltip/index.tsx:51]: @matthiasblum this is how I'm trying to solve the issue about the overgrown tooltip. It seems to work well here and in other cases , but let me know if you notice the tooltip misbehaving somewhere else

File: src/components/Description/DescriptionLLM/index.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+import Description from '..';
+import config from 'config';
+
+const css = cssBinder(globalStyles, fonts);
+
+type Props = {
+  text: string;
+};
+
+class DescriptionLLM extends PureComponent<Props> {
```
[the remark this answers is not in the document]
Comment (matthiasblum) [reply src/components/Description/DescriptionLLM/index.tsx:17]: I believe I based the component on the [`Description` one](https://github.com/ProteinsWebTeam/interpro7-client/blob/058eb658825c077cd16db20abb1bede6dec19ba8/src/components/Description/index.tsx#L20) which extends `PureComponent`.

File: src/components/Description/DescriptionLLM/index.tsx [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  render() {
+    const text = this.props.text || '';
+    if (text.length === 0) return null;
+
+    return (
+      <>
+        <div className={css('row')}>
```
[the remark this answers is not in the document]
Comment (matthiasblum) [reply src/components/Description/DescriptionLLM/index.tsx:24]: Good point. I went for the simplest solution, but indeed this needs to be changed for a better layout system.

File: src/components/Entry/Summary/index.tsx [hunk 5]
```diff
@@ -99,6 +102,38 @@ const SummaryEntry = ({
...
+        <>
+          <h4>{headerText || 'Description'}</h4>
+          <Description
+            textBlocks={metadata.description}
+            literature={included as Array<[string, Reference]>}
+            accession={metadata.accession}
+          />
+        </>
+      );
+    } else if (metadata.integrated) {
```
[the remark this answers is not in the document]
Comment (matthiasblum) [reply src/components/Entry/Summary/index.tsx:118]: Yes. Right now, we don't have any integrated signature with AI-generated descriptions, but if we ever have these cases we should prioritise descriptions provided by member databases (unless we have evidence that AI-generated descriptions are better).

File: src/components/Description/DescriptionLLM/index.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (gustavo-salazar) [reply src/components/Description/DescriptionLLM/index.tsx:17]: Right back at me!! 🤦🏽‍♂️

File: src/components/SimpleCommonComponents/Tooltip/style.css (hunk 1 above)
Comment (matthiasblum) [reply src/components/SimpleCommonComponents/Tooltip/style.css:13]: Yes. Feel free to remove it now that your fix addresses the overgrown tooltip.

**Pull request merged**